homeassistant/components/iskra/coordinator.py

```diff
@@ -0,0 +1,69 @@
+"""Coordinator for Iskra integration."""
+
+from datetime import timedelta
+import logging
+
+from pyiskra.devices import Device
+from pyiskra.exceptions import (
+    DeviceConnectionError,
+    DeviceTimeoutError,
+    InvalidResponseCode,
+    NotAuthorised,
+)
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class IskraDataUpdateCoordinator(DataUpdateCoordinator):
+    """Class to manage fetching Iskra data."""
+
+    def __init__(self, hass: HomeAssistant, device: Device) -> None:
+        """Initialize."""
+        self._device = device
+        self.name = device.serial
+        self._is_available = False
+
+        update_interval = timedelta(seconds=60)
+
+        super().__init__(
+            hass,
+            _LOGGER,
+            name=DOMAIN,
+            update_interval=update_interval,
+        )
+
+    @property
+    def available(self):
+        """Return True if entity is available."""
+        return self._is_available
+
+    @property
+    def device(self):
+        """Return the device."""
+        return self._device
+
+    async def _async_update_data(self):
```

Comment:
```suggestion
    async def _async_update_data(self) -> None:
```

Reply:
this causes error

`homeassistant/components/iskra/coordinator.py:45: error: Return type "Coroutine[Any, Any, None]" of "_async_update_data" incompatible with return type "Coroutine[Any, Any, dict[str, Any]]" in supertype "DataUpdateCoordinator"  [override]`